packages/tokens-studio-for-figma/src/plugin/updateVariables.test.ts

```diff
@@ -81,7 +80,7 @@ describe('updateVariables', () => {
     expect(result.variableIds).toEqual({ 'primary.500': 'VariableID:1:0' });
   });
 
-  it('handles tokens that reference another token', () => {
+  it('handles tokens that reference another token', async () => {
```

Comment:
```suggestion
  it('should handle tokens that reference another token', async () => {
```

Again, maybe needs a few more words to explain what 'handle' means?